### DISCUSSION TITLE
[Suggest] 한국선로세트에 고무차륜선로, 모노레일 선로 추가예정

### DISCUSSION THREAD

**Discussion by SerpensNebula** [KoreanGRF/KoreanTrainSet#472] [건의(Suggestions)]
관련된 차량들이 대응할 수 있도록 한열셋 업데이트 전에 미리 알려드립니다.

고무차륜선로 label은 KLAG, 모노레일은 KLMN 입니다.

**Comment by telk5093** [comment]
코드는 수정해봤는데, 실제 호환 여부를 확인해봐야해서 고무차륜선로, 모노레일 선로가 추가된 한국 선로 세트 개발 버전이 있다면 빌드해서 첨부해주시면 감사하겠습니다.

직접 테스트해보시려면 https://github.com/KoreanGRF/KoreanTrainSet/actions/runs/9285899766 에서 받아서 확인 가능합니다.
일단 제가 테스트해봤을 때에는 비전화 선로에서도 제3궤조 차량이 나오는 등의 문제가 있는데 이건 한국 열차 세트에서 해결이 안 될 것 같습니다.
